### ISSUE TITLE
DecoratedWindow does not update content when changing the targetState of Crossfade

### ISSUE THREAD

**Issue by Omico** [JetBrains/jewel#328]
See https://github.com/Omico/kotlin-explorer/commit/2d728aaccba6c2e38a7efc89d880d5779f878783

**Comment by rock3r** [comment]
@Omico it would be great to have a video of the issue in action and some more info such as which OS you're using

**Comment by Omico** [comment]
https://github.com/JetBrains/jewel/assets/14316223/8b932357-f8e7-45f8-a127-ec0e3e745564

**Comment by rock3r** [comment]
Thanks! It is definitely a weird issue, and I'm not sure how it relates to the decorated window — maybe it's caused by the menu bar? That looks very broken due to its placement, but I'm not sure what is the intended behaviour on Windows tbh

**Comment by Omico** [comment]
Yeah, I want to completely replace `MenuBar` with `Dropdown,` but I got stuck at https://github.com/JetBrains/jewel/issues/295#issuecomment-2016323981. Maybe I should implement a `MenuBar` entirely myself.
